### ISSUE TITLE
Set feedback for a course

### ISSUE THREAD

**Issue by Alinlish** [websquared/lfc#74]
After a student subscribed to a course, on that course, there is no option to set a feedback, or if the option is on course page, it's not easy to find it [Link](https://www.screencast.com/t/eS3PE1nnP)

**Comment by websquared** [comment]
@Alinlish this is done - please test it and close if it works fine

**Comment by Alinlish** [comment]
@websquared Yes, this option is added, but when the course is completed at 100% and clicking on Feedback [Link](https://www.screencast.com/t/TJpaOVTcGPap), one error page is displayed [Link](https://www.screencast.com/t/wlJOA2bUGJ)

**Comment by websquared** [comment]
should be fixed now

**Comment by Alinlish** [comment]
Now when completing a feedback, a comment and clicking on Submit feedback button [Link](https://www.screencast.com/t/dMzNhVHGcwRv), an error page is displayed [Link](https://www.screencast.com/t/iEHDk4LMmW3m)

**Comment by websquared** [comment]
fixed

**Comment by Alinlish** [comment]
As a student, when I buy, complete and set feedback to a course [Link](https://www.screencast.com/t/J217uPzv) and afterwards log in as instructor, the feedback is not updated [Link](https://www.screencast.com/t/BbCYBluW)

**Comment by websquared** [comment]
@Alinlish Average course rating is updated nightly - please test it the next day after submitting rating

**Comment by Alinlish** [comment]
@websquared It's implemented like you said, the average rating is updated over night.
